packages/manager/modules/sms/src/sms/batches/create/telecom-sms-batches-create.controller.js

```diff
@@ -0,0 +1,81 @@
+import moment from 'moment';
+import { SMS_COMPOSE } from '../../sms/compose/telecom-sms-sms-compose.constant';
+
+export default class SmsBatchesCreateController {
+  $onInit() {
+    this.sms = {
+      sender: this.senders[0],
+      contacts: [],
+      receivers: [],
+      immediate: true,
+    };
+  }
+
+  static canDisplayEstimation(sms) {
+    return (
+      ((sms.uniqueReceiver && sms.uniqueReceiver !== '') ||
```

Comment:
Looks like this could have been handled in a `SMS` class or at least some helpers 
(e.g : `hasReveivers(), hasContacts()` ...

Reply:
Yes it could be done with a class. Keeping that in mind in case of refactor the SMS creation page.